zeppelin-server/src/main/java/org/apache/zeppelin/realm/ZeppelinHubRealm.java

```diff
@@ -157,6 +161,9 @@ protected User authenticateUser(String requestBody) {
     ZeppelinServer.notebookWsServer.broadcastReloadedNoteList(
         new org.apache.zeppelin.user.AuthenticationInfo(account.login), userAndRoles);
 
+    // Add ZeppelinHub user_session token this singleton map, this will help ZeppelinHubRepo
+    // to get specific information about the current user.
+    UserSessionContainer.instance.setSession(account.login, userSession);
```

Comment:
i think this `setSession` should go few lines above, right after `catch` statement, since there's `broadcastReloadedNoteList` called afterwards that will require